hydra-node/json-schemas/api.yaml

```diff
@@ -1988,6 +1988,7 @@ components:
             - tag
             - headId
             - recoverTxId
```

Comment:
question: why do we need the txId if we have the utxo?

Reply:
Maybe we don't - I kept it around since more information the better I guess?